hasura.planx.uk/tests/lowcal_sessions.test.js

```diff
@@ -207,6 +300,16 @@ describe("lowcal_sessions", () => {
         expect(res.data.update_lowcal_sessions_by_pk.id).toEqual(alice1);
         expect(res.data.update_lowcal_sessions_by_pk.data).toHaveProperty("x", 1);
       });
+
+      test("Alice can update her session with an empty email", async () => {
+        const headers = {
+          "x-hasura-lowcal-session-id": alice1,
+          "x-hasura-lowcal-email": ""
+        };
+        const res = await gqlPublic(updateByPK, { sessionId: alice1, data: { x: 1 } }, headers);
+        expect(res).not.toHaveProperty("errors");
+        expect(res.data.update_lowcal_sessions_by_pk).toBeNull();
+      });
```

Comment:
This is unintended behavior but worth noting. This change means that existing sessions can be updated (but not read back) without an email.